test.mjs

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+import {runTests} from '@vscode/test-electron'
+
+await runTests({
+  extensionDevelopmentPath: new URL('.', import.meta.url).pathname,
+  extensionTestsPath: new URL('test/index.test.js', import.meta.url).pathname
+})
```

Comment:
Ideally this script will be removed entirely some day in favour of the `code-test` CLI. (microsoft/vscode-test#81)

For now using ESM and top-level await simplify it.